### PR TITLE
Fix map tooltip date

### PR DESCRIPTION
**Pull request by MarcelGerber**
Related Slack discussion: https://owid.slack.com/archives/C46U9LXRR/p1599665841022600?thread_ts=1599557290.007100&cid=C46U9LXRR

@breck7 Do you perchance know why you changed it from `targetYear` to `targetYearProp` in [this commit](https://github.com/owid/owid-grapher/commit/a57bb35880586166f1183884e09e052a5895476b#diff-99f0978524a3e523eae3da0e5cba38eeR289)?
I just want to make sure this doesn't re-surface any other issues.

Basically the problem here was that `targetYearProp` can be `Infinity` (and is in this case), and `formatYear(Infinity)` is a very helpful (and verbose!) `undefined NaN, -0NaN`

### CODE REVIEW

**Comment by breck7** [comment]
Nope. Probably just an autocomplete or global rename mistake. But there's a lot of code smell in that file :). If we designed something different for targetYear in map transforms, then we should have a test for it. Someday soon :)

**Review by breck7 — APPROVED** [review]
Ideally we should have a test for this, but Transforms are due for a cleanup so we can get them later.
